flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -62,7 +62,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-class IcebergFilesCommitter extends AbstractStreamOperator<Void>
+public class IcebergFilesCommitter extends AbstractStreamOperator<Void>
```

Comment:
We do not need this to be public

Please go through all of the files to check if we need them to make public